src/search/search.component.ts

```diff
@@ -128,6 +128,12 @@ export class Search implements ControlValueAccessor {
 	// @ts-ignore
 	@ViewChild("input", { static: false }) inputRef: ElementRef;
 
+	/**
+	 * Sets `true` when composing text via IME.
+	 */
+	isComposing = false;
+
```

Comment:
Should be protected unless there's a specific reason for it being public.

```suggestion
	protected isComposing = false;
```